6/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:xenial
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		ca-certificates       \
+		wget                  \
+                libaprutil1           \
+                libboost-regex1.58.0  \
+                libicu55              \
+                libldap-2.4-2         \
+                libtbb2               \
+                libnuma1              \
+                libxml2               \
+                libxslt1.1            \
+                g++                   \
+                openssh-client        \
+                openssh-server        \
+                expect                \
+                rsync                 \
+                python                \
+                sudo                  \
+                libblas3              \
+                libatlas3-base        \
+                psmisc                \
+                --fix-missing         \
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN groupadd hpcc && useradd -s /bin/bash -r -m -d /home/hpcc -g hpcc -c "hpcc Runtime User" hpcc
+
+ENV VERSION 6.4.16
+ENV FULL_VERSION 6.4.16-1
```

Comment:
This does not look right...

Reply:
I will update version with 6.4.20-1 is in portal and submit a PR